source/dapp.rst

```diff
@@ -3,10 +3,26 @@
 Credits Dapp
 ============
 
-Credits Dapp is a packaged Python module code that provides a
+Credits Dapp is a distributed blockchain-based application developed using
+the Credits Core framework. Dapp is a unit of business logic, and it's
+supposed to be designed with specific real life use case in mind.
+
+Credits Dapp is a software module designed to run as part of the Credits
```

Comment:
Dapp is not a module, a dapp is what I pick and choose from modules to make up MY distributed application